### PR TITLE
Use the first version found in the response.

### PR DESCRIPTION
**Pull request by albertoleal**

### CODE REVIEW

File: releases_cmd.go [hunk 1]
```diff
@@ -46,6 +46,7 @@ func (r *releasesCmd) printFiles(version string,
 	for _, release := range releases {
 		if release.Version == version {
 			files = release.UserFiles
+			break
```
Comment (pavanka) [inline releases_cmd.go]: we can definitely break if we find the version we are looking for.
nice find.

File: releases_cmd_test.go [hunk 2]
```diff
@@ -69,6 +69,12 @@ func TestReleasesCmdPrintVersin(t *testing.T) {
 			"public": {"index.html": "2", "docs.html": "2"}
 			}`,
 		},
+		{Version: "v2",
```
Comment (pavanka) [inline releases_cmd_test.go]: this does not really tests the above change.
both versions have to be "v1" and it only prints the first one.

Comment (albertoleal) [inline releases_cmd_test.go]: Hi @pavanka, there are two entries for "v2": https://github.com/albertoleal/parse-cli/blob/PrintFirstVersionFound/releases_cmd_test.go#L66-L77

Comment (pavanka) [inline releases_cmd_test.go]: oh, sure. looks good to me.
thanks for doing this :+1:

**Pull request closed without merging**